ElectronicObserver/Window/Settings/Notification/Damage/ConfigurationNotificationDamageUserControl.xaml

```diff
@@ -0,0 +1,67 @@
+﻿<UserControl
+	x:Class="ElectronicObserver.Window.Settings.Notification.Damage.ConfigurationNotificationDamageUserControl"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:base="clr-namespace:ElectronicObserver.Window.Settings.Notification.Base"
+	xmlns:controls="http://schemas.modernwpf.com/2019"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:local="clr-namespace:ElectronicObserver.Window.Settings.Notification.Damage"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	d:DataContext="{d:DesignInstance local:ConfigurationNotificationDamageViewModel}"
+	d:DesignHeight="450"
+	d:DesignWidth="800"
+	mc:Ignorable="d"
+	>
+	<Grid>
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
+			<RowDefinition />
+		</Grid.RowDefinitions>
+
+		<base:ConfigurationNotificationBaseUserControl Grid.Row="0" />
+
+		<GroupBox
```

Comment:
Hmm yeah, StackPanel and Grids has different resizing behavior. Changed the parent grid row height, so that it always has the same size as the content.